backend/handler/database/roms_handler.py

```diff
@@ -787,16 +787,15 @@ def update_rom_file(self, id: int, data: dict, session: Session = None) -> RomFi
         return session.query(RomFile).filter_by(id=id).one()
 
     @begin_session
-    def missing_rom_files(
+    def purge_rom_files(
```

Comment:
Is this change expected, or a bad rebase after Zurdi's changes?

Reply:
it's expected, we want to purge missing files since a) they dont have metadata and b) we can rescan to repopulate them